File: python chpt 3 assignments/nestedlooping.py
for row in range(10):
    for column in range(row+1):
        print('*', end='')
    print('   ', end='')

    
    for column in range(10-row):
        print('*', end='')
    print('   ', end='')

    
    for column in range(10-row):
        print(' ', end='')
    for asterik in range(row+1):
        print('*', end='')
    print('   ', end='')

    
    for column in range(row):
        print(' ', end='')
    for asterik in range(10-row):
        print('*', end='')
    print()

File: chpt 3 assign translated to java/NestedLooping.java
public class NestedLooping {
    public static void main(String[] args) {
        for (int row = 0; row < 10; row++) {
       
            for (int column = 0; column <= row; column++) {
                System.out.print("*");
            }
            System.out.print("   ");

            for (int column = 0; column < 10 - row; column++) {
                System.out.print("*");
            }
            System.out.print("   ");

            for (int column = 0; column < 10 - row; column++) {
                System.out.print(" ");
            }
            for (int asterik = 0; asterik <= row; asterik++) {
                System.out.print("*");
            }
            System.out.print("   ");

            for (int column = 0; column < row; column++) {
                System.out.print(" ");
            }
            for (int asterik = 0; asterik < 10 - row; asterik++) {
                System.out.print("*");
            }
            System.out.println();
        }
    }
}

